css-positioning/absolute.css

```diff
@@ -0,0 +1,39 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+  margin-top: 0;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 10px;
+  left: 230px;
```

Comment:
This is a _big_ movement. Instead of moving it all the way from the other side, you should try moving from the `right`